### PR TITLE
Fix for issue 243. Problem Creating schema table.

### PR DESCRIPTION
**Pull request by boedy**
https://github.com/robmorgan/phinx/issues/243

### CODE REVIEW

File: src/Phinx/Db/Adapter/PdoAdapter.php [hunk 1]
```diff
@@ -421,8 +421,8 @@ public function createSchemaTable()
 
             $table = new Table($this->getSchemaTableName(), $options, $this);
             $table->addColumn('version', 'biginteger')
-                  ->addColumn('start_time', 'timestamp')
-                  ->addColumn('end_time', 'timestamp')
+				  ->addColumn('start_time', 'timestamp', ['default' => 'CURRENT_TIMESTAMP'])
+				  ->addColumn('end_time', 'timestamp', ['default' => 'CURRENT_TIMESTAMP'])
```
Comment (robmorgan) [inline src/Phinx/Db/Adapter/PdoAdapter.php]: please fix indentation. Also this code won't work on PHP 5.3. Use `array()`

**Comment by boedy** [comment]
Why is the build failing?

**Comment by robmorgan** [comment]
@boedy see here: https://travis-ci.org/robmorgan/phinx/jobs/66756343

**Comment by shadowhand** [comment]
Thank you for contributing! The branch you made this PR against is no longer accepting patches. This probably happened because we didn't merge your changes fast enough. :sob:

Please create a new PR against the current default branch if you want to see these changes merged.

**Pull request closed without merging**